cli/src/CliProcessor.cpp

```diff
@@ -713,6 +714,10 @@ void CliProcessor::readOutputArgs(variables_map vm, bool splitArgs)
     osnArgs.dgcsCatalogID = vm.find("dgcs-catalog-id") != end(vm);
     osnArgs.evwhsCatalogID = vm.find("evwhs-catalog-id") != end(vm);
     readVariable("wfs-credentials", vm, osnArgs.wfsCredentials);
+    readVariable("extra-fields", vm, osnArgs.extraFields, true);
+    if (osnArgs.extraFields.size() > 0 && osnArgs.extraFields.size() % 2 != 0){
```

Comment:
Don't need the first check (`0 % 2==0`), you might actually extend mention the parameter, and I think the hyphen confuses the error:

```
DG_CHECK(osnArgs.extraFields.size() % 2 == 0, "Extra fields must be supplied in pairs of key and value strings.");
```

Reply:
I don't want to do anything if extraFields is empty, but I guess it's ok.